### PR TITLE
BM, CBS, aepc, pri & DHM

### PR DESCRIPTION
**Pull request by Rezy-Dev**

### CODE REVIEW

**Review by ankurgajurel — APPROVED** [review]
naice .. thanks for your contribution

**Review by ankurgajurel — APPROVED** [review]
howboutnow

**Review by ankurgajurel — APPROVED** [review]

File: data/websites.json [hunk 1]
```diff
@@ -314,6 +314,21 @@
       "name": "Bhadrapur Municipality",
       "url": "https://bhadrapurmun.gov.np/en",
       "description": "Official Website Of Bhadrapur Municipality"
+    },
+    {
+        "name": "President Office",
```
Comment (thenishantsapkota) [inline data/websites.json:319]: How about keeping the official name of the office?

File: data/websites.json [hunk 2]
```diff
@@ -314,6 +314,21 @@
...
+        "url": "https://president.gov.np/",
+        "description": "Office of the President of Nepal"
+    },
+    {
+        "name": "Policy Research Institute",
```
Comment (thenishantsapkota) [inline data/websites.json:324]: Same here
```suggest
"name": "Office of the Policy Research Institute"
```

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/websites.json [hunk 3]
```diff
@@ -316,14 +316,14 @@
...
         "url": "https://president.gov.np/",
-        "description": "Office of the President of Nepal"
+        "description": "Official Website Of the Office of the President of Nepal"
     },
     {
-        "name": "Policy Research Institute",
+        "name": "Office of the Policy Research Institute",
         "url": "https://pri.gov.np/",
-        "description": "Office of the Policy Research Institute"
+        "description": "Official Website Of the Office of the Policy Research Institute"
```
Comment (thenishantsapkota) [inline data/websites.json:326]:
```suggestion
        "description": "Official Website of the Office of the Policy Research Institute"
```

File: data/websites.json [hunk 4]
```diff
@@ -316,14 +316,14 @@
       "description": "Official Website Of Bhadrapur Municipality"
     },
     {
-        "name": "President Office",
+        "name": "Office of the President of Nepal",
         "url": "https://president.gov.np/",
-        "description": "Office of the President of Nepal"
+        "description": "Official Website Of the Office of the President of Nepal"
```
Comment (thenishantsapkota) [inline data/websites.json:321]:
```suggestion
        "description": "Official Website of the Office of the President of Nepal"
```

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

**Review by thenishantsapkota — APPROVED** [review]

**Comment by thenishantsapkota** [comment]
Thank you for your contribution @Rezy-Dev 🥳

**Pull request merged**